tools/packaging/packaging.psm1

```diff
@@ -476,6 +476,15 @@ function New-UnixPackage {
                 $AfterRemoveScript = [io.path]::GetTempFileName()
                 $packagingStrings.UbuntuAfterInstallScript -f "$Link/pwsh" | Out-File -FilePath $AfterInstallScript -Encoding ascii
                 $packagingStrings.UbuntuAfterRemoveScript -f "$Link/pwsh" | Out-File -FilePath $AfterRemoveScript -Encoding ascii
+                # Needed for Ubuntu 16.04 and Debian 8.*
+                # NOTE: Staging is ending up with the actual file instead of the symbolic link.
+                # Remove the file and create the link.
```

Comment:
Need to update the comment to explain what binary is requiring this symbolic creation. Please see the comment for creating symlink for "libssl.so" above as an example.

Reply:
I updated the comments in both packaging.psm1 and build.psm1.

Reply:
fixed